airflow/config_templates/config.yml

```
@@ -1416,6 +1416,22 @@ api:
       version_added: 2.7.0
       example: ~
       default: "False"
+    disable_regexp_middleware:
+      description: |
+        Disables the middleware that checks for regular expressions in the URL.
```

```suggestion
         A comma-separated list of paths that should be excluded from the regular expression middleware.
```
will update after seeing the K8s tests. I am having small issues with k8s test in my local. Solving in the meanwhile